quiche/src/recovery/rtt.rs

```diff
@@ -37,6 +37,8 @@ pub(crate) const RTT_WINDOW: Duration = Duration::from_secs(300);
 pub struct RttStats {
     pub(super) latest_rtt: Duration,
 
+    pub(super) max_rtt: Duration,
```

Comment:
nit: would love to avoid `pub` if possible. Do we access this outside this file?

Reply:
Done.